ui/src/components/friends/add.rs

```diff
@@ -89,12 +102,19 @@ pub fn AddFriend(cx: Scope) -> Element {
                         | Error::BlockedByUser
                         | Error::InvalidIdentifierCondition
                         | Error::CannotSendSelfFriendRequest => {
```

Comment:
The other error conditions would be irrelevant (though it looks like it been there for awhile before this PR). May be best to remove all except  `Error::CannotSendSelfFriendRequest` unless there is a reason on why those would need to be there, and if it does need to remain, then the text below should change to something else to add more context.